### DISCUSSION TITLE
Importing step file

### DISCUSSION THREAD

**Discussion by THANGABALAJI-T** [CadQuery/cadquery#1320] [Q&A]
![image](https://user-images.githubusercontent.com/125752512/233766078-19b4d6f3-2c17-44fa-a6b6-3e88a513b5bc.png)
What to do to import step file from directory?

**Comment by THANGABALAJI-T** [comment]
Even in below codes, WHAT IS THE RIGHT WAY OF GIVING THE FILE PATH : 
import cadquery as cq
result = cq.importers.importStep('/path/to/step/block.stp')

**Comment by lorenzncode** [comment]
You can try to first copy the step file to your working directory (the same location as your script or where you launched cq-editor).

Change the name to your step filename.

```py
import cadquery as cq

myfile = "mystepfile.step"

result = cq.importers.importStep(myfile)
```

If you know the path to the step file it can be provided as a string.
